packages/graphql-yoga/__tests__/integration.spec.ts

```diff
@@ -91,11 +103,11 @@ describe('Uploads', () => {
   })
   it('should upload a file', async () => {
     const UPLOAD_MUTATION = /* GraphQL */ `
-      mutation upload($file: Upload!) {
+      mutation upload($file: Blob!) {
         singleUpload(image: $file)
       }
     `
-    const response = await request(fastify.server)
+    const { body } = await request(fastify.server)
```

Comment:
Do you think we can refactor this to use the .inject method if we continue to return `Response` cause I initially did this because I was having troubles with `form-data` and fastify.inject